tests/console/verify_secure_boot.pm

```diff
@@ -0,0 +1,31 @@
+# SUSE's openQA tests
+#
+# Copyright © 2020 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Verify that secure boot is set as expected.
+# Maintainer: Sofia Syrianidou <ssyrianidou@suse.com>
+
+use strict;
+use warnings;
+use base 'y2_installbase';
+use testapi;
+use scheduler 'get_test_suite_data';
+
+sub run {
+    my $test_data = get_test_suite_data();
+    select_console 'root-console';
+    record_info("Check file", "Check if file /sys/firmware/efi/vars/SecureBoot-*/data exists");
+    assert_script_run("ls /sys/firmware/efi/vars/SecureBoot-*/data");
+    record_info("Check secure boot", "Check if secure boot option is set as expected");
+    my $is_secure = script_output("od -An -t u1 /sys/firmware/efi/vars/SecureBoot-*/data");
+    die "The secure boot option is not $test_data->{secure_boot}" if (($is_secure & ($test_data->{secure_boot} ne "enabled")) | (!($is_secure) & ($test_data->{secure_boot} ne "disabled")));
```

Comment:
make use of logical AND and not bitwise. 
With logical AND and OR you have higher precendence and you can avoid parenthesis, in contrast of && and ||.
for example:
record_info ("bla", "...") ||  die "fail!\n";
versus 
record_info "bla", "..." ||  die "fail!\n";